qbraid_qir/qasm3/oq3_maps.py

```diff
@@ -63,12 +76,25 @@ def u3_gate(builder, theta, phi, lam, qubits):
     # global phase - e^(i*(phi+lambda)/2) is missing in the above implementation
 
 
+def u3_inv_gate(builder, theta, phi, lam, qubits):
+    # reverse order of operations with negative angles
+    pyqir._native.rz(builder, -1.0 * (phi + CONSTANTS_MAP["pi"]), qubits)
+    pyqir._native.rx(builder, -1.0 * (CONSTANTS_MAP["pi"] / 2), qubits)
+    pyqir._native.rz(builder, -1.0 * (theta + CONSTANTS_MAP["pi"]), qubits)
+    pyqir._native.rx(builder, -1.0 * (CONSTANTS_MAP["pi"] / 2), qubits)
+    pyqir._native.rz(builder, -1.0 * lam, qubits)
+
+
 # Reference -
 # https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.library.U2Gate
 def u2_gate(builder, phi, lam, qubits):
     u3_gate(builder, CONSTANTS_MAP["pi"] / 2, phi, lam, qubits)
 
 
+def u2_inv_gate(builder, phi, lam, qubits):
+    u3_inv_gate(builder, CONSTANTS_MAP["pi"] / 2, phi, lam, qubits)
```

Comment:
Same thing for these two, either Python typehints or a short docstring to describe input/output types would be helpful.